Knowledge-Base/Virtiofs:-Shared-file-system.md

```diff
@@ -119,7 +128,12 @@ Also, parameters named `OverflowUid` and `OverflowGid` are always parsed from th
 
 ### Case insensitivity
 
-*At the moment, the case insensitivity feature is under development and may not work properly in some cases.* Case insensitivity can be turned on by `-i` command-line parameter of `CaseInsensitive` registry key. In this situation, when processing a request, virtiofs first tries to access the file or directory by its exact name, as in case-sensitive mode. Second, if there is no such node, virtiofs finds a node with a matching name in the parent directory, ignoring case.
+*At the moment, the case insensitivity feature is under development and may not work properly in some cases.*   
```

Comment:
I think the line “*At the moment, the case insensitivity feature is under development and may not work properly in some cases.* “ should be removed.

The feature exists and it works in specific way that you described. If someone will wants to improve it, they are welcomed, but for now we consider it is operational.